[RFC] man.vim: doc fixes

nhooyr opened the pull request:
- Weird tab+space combination used for alignment. All spaces now
- Added back `<C-T>` mapping (somehow we missed that completely)
- Fixed mistake that `<Plug>(Man)` opens in a new tab. Also added note at
  top on how the window is chosen/opened.
- Clarified q local mapping
- Removed section that shows an example autocmd to add desired folding
  style.
- Removed random line in `usr_12.txt` about `<Leader>` and backslash.

jamessan commented on runtime/doc/filetype.txt:
```diff
@@ -513,40 +513,40 @@ Local mappings:
      export MANPAGER="nvim -c 'set ft=man' -"
 
+All commands and mappings will attempt to reuse the closest man window
+(left/right) and otherwise create a horizontal split.
```
I would leave out the word "horizontal" here, since that's the default.  Also, once the `<mods>` patch gets included from Vim, then the command should be updated to use that so the user could do `:vert Man ...` to get a vertical split.

jamessan commented on runtime/doc/filetype.txt:
```diff
@@ -513,40 +513,40 @@ Local mappings:
 
 View manpages in Nvim. Supports highlighting, autocompletion, locales, and
-navigation. See also |find-manpage|.
+navigation. Also see |find-manpage|.
```
"See also" is pretty typical wording.  Either works, but I would wager that "see also" is more common.

nhooyr replied:
"Also see" flows better imo. But whatever, I'll revert.

nhooyr replied:
I didn't even notice but "Also see" is used throughout that file, so it fits in better. I'm gonna leave it.

jamessan replied:
Indeed, "also see" is used almost 4x more than "see also" in runtime/doc/.

jamessan commented on runtime/doc/filetype.txt:
```diff
@@ -513,40 +513,40 @@ Local mappings:
+Man {name}                Display the manpage for {name}.
+Man {sect} {name}         Display the manpage for {name} and section {sect}.
+Man {name}({sect})        Alternate syntax which auto-completes the section.
+Man {sect} {name}({sect}) Used during auto completion to show the real section of
```
s/auto //.  There is no auto-completion in vim.  It's all triggered.

jamessan commented on runtime/doc/filetype.txt:
```diff
@@ -513,40 +513,40 @@ Local mappings:
+                          but otherwise :close.
 
 Variables:
-g:no_man_maps		  Do not create mappings in manpage buffers.
```
Should there be tags added for these variables?

nhooyr replied:
I see no reason not.

jamessan commented on runtime/doc/filetype.txt:
```diff
@@ -513,40 +513,40 @@ Local mappings:
+All commands and mappings will attempt to reuse the closest man window
+(left/right) and otherwise create a horizontal split.
+
 Commands:
```
Add a tag for `:Man`?

nhooyr replied:
Isn't there already one at the top?

```
MAN					*ft-man-plugin* *:Man* *man.vim*
```

jamessan replied:
Guess I'd expect the tag for the command to bring you to the help for the command, not to the top of the file containing the command.

nhooyr committed:
man.vim: doc fixes

- Weird tab+space combination used for alignment. All spaces now
- Added back <C-T> mapping (somehow we missed that completely)
- Fixed mistake that <Plug>(Man) opens in a new tab. Also added note at
  top on how the window is chosen/opened.
- Clarified q local mapping
- Removed section that shows an example autocmd to add desired folding
  style.
- Removed random line in `usr_12.txt` about `<Leader>` and backslash.
- :Man supports completion, not auto-completion.

nhooyr committed:
man.vim: update docs for command modifiers

justinmk commented:
Merged with some tweaks.
